src/persist-localstorage-experimental/index.ts

```diff
@@ -20,6 +21,12 @@ interface Options {
   /** A unique string that can be used to forcefully
    * invalidate existing caches if they do not share the same buster string */
   buster?: string
+  /** A list of QueryKeys to be blacklisted and prevented from persisted
+   * Example: you dont wanna persist an access token here especially we persisting without
+   * any sort of encryption */
+  blacklistQueries?: QueryKey[];
+  /** A list of MutationKeys to be blacklisted and prevented from persisted */
+  blacklistMutations?: MutationKey[];
```

Comment:
We should also be avoiding charged terminology like blacklist. A good alternative might be disallow Queries, filter Queries etc.

Reply:
I think `blocklist` / `allowlist` ist the new `blacklist` / `whitelist`